packages/passport/sdk/src/Passport.ts

```diff
@@ -198,7 +200,11 @@ export class Passport {
         logger.warn('Failed to retrieve a cached user session', error);
       }
       if (!user && !useCachedSession) {
-        user = await this.authManager.login(options?.anonymousId);
+        if (silent) {
+          user = await this.authManager.forceUserRefresh();
```

Comment:
If the user has an expired application session and an expired refresh token, then this will trigger two requests to use the refresh token (one when `this.refreshTokenAndUpdatePromise()` is called from `getUser()`, and one in `forceUserRefresh`)

Reply:
I would be the same case for the current login, right?
`await this.authManager.login(options?.anonymousId);`